extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -340,10 +389,38 @@ export class SystemLayer {
    * @private
    */
   onMutedStateUpdate_(isMuted) {
+
     this.vsync_.mutate(() => {
-      isMuted ?
-        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, '') :
-        this.getShadowRoot().removeAttribute(AUDIO_MUTED_ATTRIBUTE);
+      if (isMuted) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'muted');
+      } else if (this.storeService_.get(StateProperty.PAGE_HAS_AUDIO_STATE)) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'audioon');
+      } else {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'noaudio');
+      }
+
+    });
+  }
+
+  /**
+   * Hides element after elapsed time
+   * @param {int} hideTimeout
+   * @private
+   */
+  hideAfterTimeout_(hideTimeout) {
+    this.timer_.delay(() => this.hideInteral_(), hideTimeout);
+  }
+
+  /**
+   * Hides message
+   * @private
+   */
+  hideInteral_() {
```

Comment:
Nit: what do you think of using something more descriptive as a method name here, so it's easier to know what's hidden by this method? Maybe something like `hideAudioMessage_` and `hideAudioMessageAfterTimeout_`